deployment/deploy.sh

```diff
@@ -8,14 +8,19 @@ IMG=$ACR.azurecr.io/neglected-diagnostics
 
 set -e
 # Build a local wheel to help with installing dependencies in the container. See Dockerfile
+rm -rf dist/*.whl
 python -m build --wheel .
 # Can be used to build/run locally:
 # docker build -t ndiag --platform=linux/arm64 -f deployment/Dockerfile .
 # docker run --rm -p 127.0.0.1:8501:80 ndiag
 
 # ACR build is preferrably to docker build since it builds in azure and avoids having to build cross-platform locally
+az account set --subscription "Carlos Garcia Jurado Suarez_SSEC" # Use SSEC subscription
```

Comment:
Do you want your name directly here or can this be an env variable?

Reply:
Good idea, I'll move to an env var. Thx!